src/drivers/websql.js

```diff
@@ -202,16 +202,16 @@ function _setItem(key, value, callback, retriesLeft) {
                 } else {
                     dbInfo.db.transaction(function(t) {
                         tryExecuteSql(t, dbInfo,
-                            `INSERT OR REPLACE INTO
+                                      `INSERT OR REPLACE INTO
                              ${dbInfo.storeName}
                              (key, value) VALUES (?, ?)`,
-                            [key, value],
-                            function() {
-                                resolve(originalValue);
-                            },
-                            function(t, error) {
-                                reject(error);
-                            }
+                                      [key, value],
```

Comment:
The indentation here (multiline backtick insert statement)  seems a bit snaky/misaligned !?
![screenshot-2017-9-24 feat localforage implement dropinstance by tofumatt pull request 741 localforage localforage](https://user-images.githubusercontent.com/1295829/30783148-d410c98a-a146-11e7-9a69-768cf3d0cec2.png)

Was that intentional? Should I fix that?
I've also lately experimented with [prettier](https://github.com/prettier/prettier) which worked great as a precommit hook for code formatting.

Reply:
Oh cool! I'm a bit lazy when it comes to setting up pre-commit hooks but others have recommended prettier too. It seems neato.

That wasn't intentional, I think it might have happened by mistake when I fixed most issues using `eslint --fix`. Fine to change it 😄